### PR TITLE
Set branch to fetch config from in env file

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

**Review by ShawTim — APPROVED** [review]

**Comment by danieledamiani** [comment]
in review

File: packages/tempus-client_v3/src/config/ConfigManager.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ const TempusPoolsConfig = {
   owner: 'tempus-finance',
   repo: 'tempus-pools-config',
   path: 'config.json',
+  ref: process.env.REACT_APP_CONFIG_BRANCH,
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/config/ConfigManager.ts:8]: If `ref` is `undefined` does it revert to `master` or should it be explicited?

**Review by danieledamiani — APPROVED** [review]
one comment

File: packages/tempus-client_v3/src/config/ConfigManager.ts (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/config/ConfigManager.ts:8]: Yes, default branch is branch that is set as a default in target repo (in this case master).

**Pull request merged**